### PR TITLE
chore: formatted files based on prettier, eslint & stylelint config

### PR DESCRIPTION
**Pull request by debjyoti-pandit**
This PR contains only formating changes based on prettier rules configured.

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 yarn lint
+yarn pretty-quick --staged --no-restage
```
Comment (SanjalKatiyar) [inline .husky/pre-commit:5]: add this command to package.json and then use that here...

**Comment by SanjalKatiyar** [comment]
thanks for the fix !

File: package.json [hunk 2]
```diff
@@ -65,6 +64,7 @@
     "css-loader": "0.28.x",
     "eslint": "^7.31.0",
     "eslint-config-prettier": "^8.3.0",
+    "eslint-import-resolver-typescript": "^2.7.1",
```
Comment (SanjalKatiyar) [inline package.json:67]: is this import for ` 'plugin:import/typescript',`

File: .eslintrc.js [hunk 3]
```diff
@@ -1,63 +1,72 @@
...
-            },
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  plugins: ['react', '@typescript-eslint', 'import'],
+  rules: {
+    'import/no-unresolved': 'off',
+    'import/extensions': 'off',
+    'import/prefer-default-export': 'off',
+    'react/prop-types': 'off',
```
Comment (SanjalKatiyar) [inline .eslintrc.js:26]:
```suggestion
    'import/no-named-as-default': 'off',
    'react/prop-types': 'off',
```

File: package.json (hunk 2 above)
Comment (debjyoti-pandit) [reply package.json:75]: yes, it is for that only [ref](https://github.com/import-js/eslint-plugin-import#typescript)

**Pull request merged**